src/register/RegistrationPage.jsx

```diff
@@ -412,10 +412,19 @@ const RegistrationPage = (props) => {
   };
 
   const handleOnBlur = (event) => {
+    const { name, value } = event.target;
     if (registrationEmbedded) {
```

Comment:
can't we do something like
```suggestion
    if (registrationEmbedded && name != 'name') {
```